File: src/main/java/com/itsci/mju/maebanjumpen/partyrole/service/PartyRoleService.java
package com.itsci.mju.maebanjumpen.partyrole.service;

import com.itsci.mju.maebanjumpen.partyrole.dto.PartyRoleDTO;

import java.util.List;

public interface PartyRoleService {
    PartyRoleDTO savePartyRole(PartyRoleDTO partyRoleDto);
    PartyRoleDTO getPartyRoleById(int id);
    List<PartyRoleDTO> getAllPartyRoles();
    PartyRoleDTO updatePartyRole(int id, PartyRoleDTO partyRoleDto);
    void deletePartyRole(int id);
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/partyrole/service/PartyRoleService.kt
package com.itsci.mju.maebanjumpen.partyrole.service

import com.itsci.mju.maebanjumpen.partyrole.dto.PartyRoleDTO

interface PartyRoleService {
    fun savePartyRole(partyRoleDto: PartyRoleDTO): PartyRoleDTO
    fun getPartyRoleById(id: Int): PartyRoleDTO?
    fun getAllPartyRoles(): List<PartyRoleDTO>
    fun updatePartyRole(id: Int, partyRoleDto: PartyRoleDTO): PartyRoleDTO
    fun deletePartyRole(id: Int)
}

